cypress/e2e/article.cy.js

```diff
@@ -1,24 +1,184 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
 
-describe('Article', () => {
-  before(() => {
+import { generateNewArticle } from "../support/generateNewArticle";
+import faker from 'faker';
 
+describe('Create new article', () => {  
+  beforeEach(() => {
+    cy.task('db:clear');
+    cy.registerNewUser().then(user => {
+      cy.login(user).then(() => user);
+    }).as('user');
+
+    cy.visit(`#/editor`);
+  });
+
+  it(`The Creating new article page 
+    should contains 'Article Title' field`, () => {
+      cy.findByPlaceholder('Article Title').should('exist');
+  });
+
+  it(`The Creating new article page 
```

Comment:
Too long page name, try to make it shorter. Check everywhere

Reply:
Done